src/main/java/duke/commands/CommandResult.java

```diff
@@ -0,0 +1,17 @@
+package duke.commands;
+
+/**
+ * Represents the result of a command execution.
+ */
+public class CommandResult {
+
+    private final String feedbackToUser;
+
+    public CommandResult(String feedbackToUser) {
+        this.feedbackToUser = feedbackToUser;
+    }
+
+    public String getFeedbackToUser() {
+        return feedbackToUser;
+    }
```

Comment:
As these 2 methods are public, may be you can consider adding JavaDocs for them.